xmonad-contrib.cabal

```diff
@@ -318,6 +318,7 @@ library
                         XMonad.Prompt.Zsh
                         XMonad.Util.ActionCycle
                         XMonad.Util.ClickableWorkspaces
+                        XMonad.Util.ClickableWorkspaces.Integrations
```

Comment:
I wonder what the rationale is for creating a whole different module for these?  Seems like they would fit in `XMonad.Util.ClickableWorkspaces` just fine

Reply:
Probably premature and entirely unnecessary cyclic deps prevention. I couldn't decide whether to put these into ClickableWorkspaces or into IndependentScreen and WorkspaceNames, and just added a separate module instead. I can probably put these into ClickableWorkspaces if I'm the only one who thought it's somehow wrong. :-)